
#include "centre_player_scene_response_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void CentrePlayerSceneReply::EnterScene(entt::entity player,const ::CentreEnterSceneRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerSceneReply::LeaveScene(entt::entity player,const ::CentreLeaveSceneRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerSceneReply::LeaveSceneAsyncSavePlayerComplete(entt::entity player,const ::CentreLeaveSceneAsyncSavePlayerCompleteRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerSceneReply::SceneInfoC2S(entt::entity player,const ::CentreSceneInfoRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}

